index.html

```diff
@@ -0,0 +1,169 @@
+        fill="#979493" />
+    </svg>
+  </section>
+  <!-- Hamburguer menu elements-->
+
+
+
+
+    <section class="first__section">
+      <div class="rocket__global">
+        <h3>"Hello Rocket world!"</h3>
+        <h1>ROCKET GLOBAL SUMMIT 2023</h1>
+      </div>
+      <p>A joyful celebration believing in the value of engineering, creating a positive change with people from over 80
+        countries is taking place in October, in Colombia.</p>
+      <div>
+        <h2 class="rocket__global__date">2023.10.17(THU) ~ 20(FRI)</h2>
+        <h3 class="rocket__global__place">Corferias Main Room, National University and more
+        </h3>
+      </div>
+
+    </section>
+  </div>
+  <section id="program" class="main__program">
+    <h1>Main Program</h1>
+    <div class="main__line"></div><!-- orange underline-->
+    <section class="main__program__cards">
+      <div class="card">
+        <img src="./images/lecture.png" alt="Lecture">
+        <h3>Lecture</h3>
+        <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
+      </div>
+      <div class="card">
+        <img src="./images/exhibition.png" alt="Exhibition">
+        <h3>Exhibition</h3>
+        <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi</p>
+      </div>
+      <div class="card">
+        <img src="./images/forum.png" alt="Forum">
+        <h3>Forum</h3>
+        <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+      </div>
+      <div class="card">
+        <img src="./images/workshop.png" alt="Workshop">
+        <h3>Workshop</h3>
+        <p>Try creating your own work using open source license rather than just watching at it.</p>
+      </div>
+      <div class="card">
+        <img src="./images/ignite.png" alt="Lecture">
+        <h3>Ignite</h3>
+        <p>Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+      </div>
+    </section>
+    <button class="join__rocket desktop__hide" type="button">Join Rocket Global Summit 2023</button>
+    <a class="see__program hide desktop__show">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section id="speakers">
+    <h1>Featured Speakers</h2>
+    <div class="main__line"></div>
+    <section class="featured__speakers">
+
+    </section>   
+
+  </section>
+  <section id="partner" class="partner hide desktop__show">
+    <h1>Partner</h1>
+    <div class= "line"></div>
+    <img src="./images/partner_desktop.png" alt="Partner">
+  </section>
+  <footer class="hide desktop__show">
```

Comment:
- [ ] I notice that your footer mobile version is bit different from the orignal design . Please make sure that it folow exactly the mobile version of the original design in order to give a good user experience and have a beautifull design  :+1:

Reply:
![image](https://user-images.githubusercontent.com/93690227/211180898-652d63ef-6f9b-414d-bcc2-6f95f451530a.png)
Thanks for your review, I checked the guide file and there is no footer for the mobile version on the home page, the last thing seen in the figure is the more button.